messaging/rabbitmq_queue.py

```diff
@@ -0,0 +1,257 @@
+            - rabbitMQ user for connection
+        required: false
+        default: guest
+    login_password:
+        description:
+            - rabbitMQ password for connection
+        required: false
+        default: false
+    login_host:
+        description:
+            - rabbitMQ host for connection
+        required: false
+        default: localhost
+    login_port:
+        description:
+            - rabbitMQ management api port
+        required: false
+        default: 15672
+    vhost:
+        description:
+            - rabbitMQ virtual host
+        required: false
+        default: "/"
+    durable:
+        description:
+            - whether queue is durable or not
+        required: false
+        choices: [ "yes", "no" ]
+        default: yes
+    autoDelete:
+        description:
+            - if the queue should delete itself after all queues/queues unbound from it
+        required: false
+        choices: [ "yes", "no" ]
+        default: no
+    messageTTL:
+        description:
+            - How long a message can live in queue before it is discarded (milliseconds)
+        required: False
+    autoExpires:
+        description:
+            - How long a queue can be unused before it is automatically deleted (milliseconds)
+        required: false
+    maxLength:
+        description:
+            - How many messages can the queue contain before it starts rejecting
+        required: false
+    deadLetterExchange:
+        description:
+            - Optional name of an exchange to which messages will be republished if they
+            - are rejected or expire
+        required: false
+    deadLetterRoutingKey:
+        description:
+            - Optional replacement routing key to use when a message is dead-lettered.
+            - Original routing key will be used if unset
+        required: false
+    arguments:
+        description:
+            - extra arguments for queue. If defined this argument is a key/value dictionary
+        required: false        
+'''
+
+EXAMPLES = '''
+# Create a queue
+- rabbitmq_queue: name=myQueue
+
+# Create a queue on remote host
+- rabbitmq_queue: name=myRemoteQueue login_user=user login_password=secret login_host=remote.example.org
+'''
+
+import requests
+import urllib
+import json
+
+def main():
+    module = AnsibleModule(
+        argument_spec = dict(
+            state = dict(default='present', choices=['present', 'absent'], type='str'),
+            name = dict(required=True, type='str'),
+            login_user = dict(default='guest', type='str'),
+            login_password = dict(default='guest', type='str', no_log=True),
+            login_host = dict(default='localhost', type='str'),
+            login_port = dict(default='15672', type='str'),
+            vhost = dict(default='/', type='str'),
+            durable = dict(default=True, choices=BOOLEANS, type='bool'),
+            autoDelete = dict(default=False, choices=BOOLEANS, type='bool'),
```

Comment:
I actually noticed the same thing when I copied the login_* syntax from the mysql modules. I've done this change later when i figured this might actually be useful to share to other people . Since there were already other modules using this syntax i wanted to preserve it as it was instead of changing to camel case.

I've done a quick search now on some modules and it seems they all follow that syntax instead of camel case (which i personally prefer). Consistency in behavior makes all sense and as such I'll be making the changes later and submit.